packages/jsii-pacmak/lib/targets/dotnet/dotnetgenerator.ts

```diff
@@ -103,6 +106,27 @@ export class DotNetGenerator extends Generator {
     await fs.mkdirp(path.join(outdir, packageId));
     await fs.copyFile(tarball, path.join(outdir, packageId, tarballFileName));
 
+    // Attempt to download the package icon from the configured URL so we can use the non-deprecated PackageIcon
+    // attribute. If this fails or is opted out (via $JSII_PACMAK_DOTNET_NO_DOWNLOAD_ICON being set), then only the
+    // deprecated PackageIconUrl will be emitted.
+    const iconFile =
```

Comment:
Is there a way for the user to pass this file without downloading it (eg if they already have the icon locally)?

Reply:
Not currently... That'd require they bundle the icon in their npm package, which I'm not sure is appropriate...